src/pool/srv_pool.c

```diff
@@ -7114,6 +7134,26 @@ ds_pool_target_status_check(struct ds_pool *pool, uint32_t id, uint8_t matched_s
 	return target->ta_comp.co_status == matched_status ? 1 : 0;
 }
 
+int
+ds_pool_mark_connectable(struct ds_pool_svc *ds_svc)
+{
+	struct pool_svc		*svc = pool_ds2svc(ds_svc);
+	struct rdb_tx		 tx;
+	int			 rc;
+
+	rc = rdb_tx_begin(svc->ps_rsvc.s_db, svc->ps_rsvc.s_term, &tx);
+	if (rc != 0) {
```

Comment:
[Requesting changes] `rc == 0`? Normally we would write

```
if (rc != 0)
        handle the error;
handle the normal case;
```

so typos like this would be caught easily.

Reply:
Sorry for the typo, fix it.